redbot/cogs/alias/alias.py

```diff
@@ -287,22 +288,62 @@ async def _del_global_alias(self, ctx: commands.Context, alias_name: str):
 
     @alias.command(name="list")
     @commands.guild_only()
+    @checks.bot_has_permissions(add_reactions=True)
     async def _list_alias(self, ctx: commands.Context):
         """List the available aliases on this server."""
         guild_aliases = await self._aliases.get_guild_aliases(ctx.guild)
         if not guild_aliases:
             return await ctx.send(_("There are no aliases on this server."))
-        names = [_("Aliases:")] + sorted(["+ " + a.name for a in guild_aliases])
-        await ctx.send(box("\n".join(names), "diff"))
+        names = [""] + sorted(["+ " + a.name for a in guild_aliases])
+        message = ""
+        for a in names:
+            message += "{}\n".format(a)
```

Comment:
This can be simplified by using `.join()` as it were before the change. This will also get rid of the trailing `\n` that makes the last page look different.
```suggestion
        message = "\n".join(names)
```